src/dodona/feedback/CloseTest.java

```diff
@@ -1,13 +1,21 @@
 package dodona.feedback;
 
+import static dodona.junit.JUnitJSON.PROPERTY_OUTPUT_CUTOFF;
+
 public class CloseTest extends AcceptedClose {
 
+    private static final int output_cutoff = Integer.getInteger(PROPERTY_OUTPUT_CUTOFF, 1000);
```

Comment:
Shouldn't we move the default value out of the Java code, into the `run`-script? I am aware that this value will not be changed often, but it would make sense to store the default values of all configurable properties in one place.

Reply:
I'll do this and add a warning here. Reason I picked this place was because the code needs a check here, anyway.